### ISSUE TITLE
JSN-SR04

### ISSUE THREAD

**Issue by diverbill** [alaudet/raspi-sump#72]
Hello,
I have been using raspi-sump for a few months and decided to add a second one for my yard sump. Unfortunately the sensor failed due to the moist conditions so i am trying to set up this new sensor and hooked it up as the old sensor was. Are there other resistor needed for this model of sensor or is it a different wiring setup?

**Comment by alaudet** [comment]
SR04T right?  Should be the same, as long as it has 4 pins.  One caveat with this sensor is that it does not work for readings closer than 20cm's.  Other than that it should be fine.

**Comment by diverbill** [comment]
Great, I thought that might be the case however the first time I could not get a reading, so a very quickly made a short cable to test with. I got it to work with one of the other sensors (the one you originally recommended) so the pi and cables checked out so I wired up the new sensor and the board got warm I'll grab some photos in the morning. 

This is what I got:

2Pcs JSN-SR04T Integrated... https://www.amazon.com/dp/B08T5XF6Y9?ref=ppx_pop_mob_ap_share

Thank you for replying so quickly!
